lib/y2lan_restart_common.pm

```diff
@@ -149,14 +149,23 @@ sub check_network_status {
     }
     assert_script_run 'cat journal.log';                              # print journal.log
     my $is_empty = script_run('[ -s journal.log ]');
+    # if journal.log is empty implies that we could not find logs indicating that the services is not restarted or reloaded
     if ($is_empty) {
-        assert_equals($expected_status, 'no_restart', "Network restart didn't occur, even though it was expected.");
+        assert_equals($expected_status, 'no_restart', "Network service should have been restarted/reloaded. journalctl cannot find events");
     }
+    # if journal.log contains one of the shutting down|ifdown all|Stopping wicked then it is assumed that the service is restarted otherwise it is reloaded.
```

Comment:
let's avoid mentioning exact query in comments . Imagine tomorrow someone will update query but will overlook it in comments . This will create huge confusion.

Reply:
changed